docs/tutorial/fdfitting.rst

```diff
@@ -380,82 +439,59 @@ Note that this piece of code will lead to parameters `DNA/Lc_0`, `DNA/Lc_1` etc.
+Let’s say we have two models, `model1` and `model2` and we want to fit both in a global fit.
+The first step is to constructing the :class:`~lumicks.pylake.FdFit`::
 
     model1 = lk.ewlc_odijk_force("DNA")
     model2 = (lk.ewlc_odijk_distance("DNA") + lk.ewlc_odijk_distance("protein")).invert()
     fit = lk.FdFit(model1, model2)
 
-But then the question arises, how do we add data to each model? Well, the trick is in the assignments to `model1` and
-`model2`. We can use these now to add data to each model as follows::
+First, we add fd data from before the unfolding event from two different measurements:::
 
-    fit[model1].add_data("data for model 1", forces_1, distances_1)
-    fit[model2].add_data("data for model 2", forces_2, distances_2)
+    for i, (distance, force) in enumerate(zip(distances, forces)):
+        fit[model1].add_data(f"Before unfolding {i}", force, distance)
 
 See how we used the model handles? They are used to let the :class:`~lumicks.pylake.FdFit` know to which model each data set should be added.
-You can add as many data sets as you want to both models and fit it all at once.
-
-Plotting is straightforward in this setting. We can plot the data sets corresponding to model 1 and 2 as follows::
-
-    fit[model1].plot()
-    fit[model2].plot()
-
-Accessing the model parameters for a specific data set is a little more complicated in this setting. If we want to
-obtain the parameters for "data for model 1", we'd have to invoke::
+Rather then directly adding the data from after the unfolding event, and fitting everything together, we are going to fit in incrementally::
 
-    params = fit[model1]["data for model 1"]
 
-Note how we are now forced to index the model first using the square brackets, and only then access the data set by
-name. An unfortunate necessity when it comes to multi-model curve fitting.
+    fit.fit()
 
+Next, add data after unfolding to `fit[model2]`::
 
-Confidence intervals and standard errors
-****************************************
+    d_adk1 = file_adk1.fdcurves['adk5_curve1'].d['21s':'29s'].data
+    f_adk1 = file_adk1.fdcurves['adk5_curve1'].f['21s':'29s'].data
 
-Once parameters have been fitted, standard errors can easily be obtained as follows::
+    d_adk2 = file_adk2.fdcurves['adk5_curve2']['20s':'29s'].d.data
+    f_adk2 = file_adk2.fdcurves['adk5_curve2']['20s':'29s'].f.data
 
-    fit["DNA/Lc"].stderr
+    distances2 = [d_adk1, d_adk2]
+    forces2 = [f_adk1, f_adk2]
 
-Assuming that the parameters are not at the bounds, the sum of random variables with finite moments converges to a
-Gaussian distribution. This allows for the computation of confidence intervals using the Wald test
-:cite:`press1990numerical`. To get these asymptotic intervals, we can use the member function `.ci` with a desired
-confidence interval::
+    for i, (distance, force) in enumerate(zip(distances2, forces2)):
+        fit[model2].add_data(f"After unfolding {i}", force[distance>0], distance[distance>0])
```

Comment:
There's data with negative distance? Is this due to noise, or does the average distance actually go negative?

Reply:
The distance is actually always positive (the >0 part was copy-pasted from the AdK Notebook on Harbor). I removed  ' `[distance>0]` '

Reply:
Update: The distance is always positive _or_ 0. So the >0 statement was there to remove the distance = 0 datapoints, resulting from bad camera tracking. I put the >0 back, but at a different location, right when `distances` and `forces` are defined.

Reply:
Right, that makes sense. But it should probably be mentioned in the text somewhere, since it's something that may surprise someone.